model/patch/db_test.go

```diff
@@ -75,7 +75,145 @@ func TestMostRecentByUserAndProject(t *testing.T) {
 	assert.NotNil(t, p)
 	assert.Equal(t, p.Id, previousPatch.Id)
 }
+func TestByPatchNameStatusesCommitQueuePaginatedRequestersOption(t *testing.T) {
+	for tName, tCase := range map[string]func(ctx context.Context, t *testing.T){
+		"EmptyRequestersList": func(ctx context.Context, t *testing.T) {
+			opts := ByPatchNameStatusesCommitQueuePaginatedOptions{
+				Project:    utility.ToStringPtr("evergreen"),
+				Requesters: []string{evergreen.PatchVersionRequester},
+			}
```

Comment:
should the test be called `EmptyRequestersList`? You also have `NoRequestersList` below which covers the empty case I think